Tests/Common/SymbolTests.cs

```diff
@@ -270,6 +269,22 @@ public void ComparesAgainstStringIgnoringCase()
             Assert.AreEqual(0, a.CompareTo("A"));
         }
 
+        [Test]
+        public void ComparesAgainstNull()
+        {
+            var validSymbol = Symbols.SPY;
+            var emptySymbol = Symbol.Empty;
+            Symbol nullSymbol = null;
+
+            Assert.IsTrue(validSymbol != null);
+            Assert.IsTrue(emptySymbol != null);
```

Comment:
I think we want the empty symbol to behave just like null, it's a 'null impl' of the symbol type. Conceptually `Symbol.Empty` and `(Symbol)null` are exactly the same -> the symbol wasn't specified, so I think they should compare the same

Reply:
If we want `Symbol.Empty` to behave like `(Symbol)null`, don't we just need to change its definition ?
from:
        `public static readonly Symbol Empty = new Symbol(SecurityIdentifier.Empty, string.Empty);`
to:
        `public static readonly Symbol Empty = null;`